src/stores/room-list/filters/VisibilityProvider.ts

```diff
@@ -54,6 +55,11 @@ export class VisibilityProvider {
             return false;
         }
 
+        const cli = MatrixClientPeg.get();
+        if (cli.getIgnoredInvites()?.ignored_rooms?.includes(room.roomId) ?? false) {
```

Comment:
surely this should also be checking the room is an invite, in case you joined it via a client which doesn't support that MSC. This specifically says `getIgnoredInvites`

Reply:
yes, this is true, but there's ongoing discussion about whether we just want to ignore invites, or ignore rooms wholesale; the idea being someone might want to ignore a room they're already in. i should have made that clear in the body of this PR

Reply:
I'm surprised this is the shape it is, ignoring users happens at the server level, why wouldn't ignoring rooms work the same? With the way it is currently structured, your /sync may include a billion rooms which you ignore at the UI level.

Reply:
I believe that we should focus on ignoring invites for the time being.